proxysql/manifest.json

```diff
@@ -27,7 +27,9 @@
     "configuration": {
       "spec": "assets/configuration/spec.yaml"
     },
-    "dashboards": {},
+    "dashboards": {
+      "ProxySQL - Overview": "assets/dashboards/-_overview.json"
```

Comment:
This path seems not nice `assets/dashboards/-_overview.json`. You an avoid that by changing dashboard name to `ProxySQL Overview`.